node/test/Test/Pos/CryptoSpec.hs

```diff
@@ -481,18 +471,19 @@ keygenInequality a b =
 goodShareIsDecrypted :: Crypto.VssKeyPair -> Crypto.EncShare -> Property
 goodShareIsDecrypted vssKP encShare = monadicIO $ do
     decShare <- run $ Crypto.decryptShare vssKP encShare
-    assert $ Crypto.verifyShare encShare (Crypto.toVssPublicKey vssKP) decShare
+    assert $ Crypto.verifyDecShare (Crypto.toVssPublicKey vssKP) encShare decShare
 
+{-
 verifyEncShareGoodData :: SharedSecrets -> Bool
 verifyEncShareGoodData SharedSecrets {..} =
     Crypto.verifyEncShare ssSecShare
                           (ssVSSPKs !! ssPos)
                           (fst $ ssShares !! ssPos)
 
-verifyEncShareBadSecShare :: SharedSecrets -> Crypto.SecretSharingExtra -> Property
-verifyEncShareBadSecShare SharedSecrets {..} secShare =
-    (ssSecShare /= secShare) ==>
-    not $ Crypto.verifyEncShare secShare (ssVSSPKs !! ssPos) (fst $ ssShares !! ssPos)
+verifyEncShareBadSecShare :: SharedSecrets -> Crypto.SecretProof -> Property
```

Comment:
Shouldn't it be just removed?

Reply:
It might be possible to resurrect it. I'm going to try now.

Reply:
Okay, I resurrected this test and will resurrect the others too.